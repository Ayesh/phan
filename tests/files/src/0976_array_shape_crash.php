<?php
$x = ["\x80" => 42];
if (random_int(0, 1) === 0) {
    $x = null;
}
$y = $x["\x80"] ?? null;
'@phan-debug-var $y';
