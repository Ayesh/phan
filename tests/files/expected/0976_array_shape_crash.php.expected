%s:6 PhanDebugAnnotation @phan-debug-var requested for variable $y - it has union type ?42
